drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -465,6 +461,11 @@ void at86rf2xx_set_state(at86rf2xx_t *dev, uint8_t state)
         old_state = at86rf2xx_get_status(dev);
     }
 
+    if (state == AT86RF2XX_STATE_FORCE_TRX_OFF) {
+        _set_state(dev, AT86RF2XX_STATE_TRX_OFF, state);
+        return;
+    }
+
```

Comment:
I can't see how moving this block helps. `at86rf2xx_configure_phy` uses `AT86RF2XX_STATE_TRX_OFF` not `AT86RF2XX_STATE_FORCE_TRX_OFF` thus it shouldn't have an effect.